protos/sources/vega/events/v1/events.proto

```diff
@@ -105,8 +107,8 @@ message FeesStats {
   repeated PartyAmount total_maker_fees_received = 8;
   // Maker fees paid by all trade aggressors, and which makers the fees were paid to.
   repeated MakerFeesGenerated maker_fees_generated = 9;
-  // All trading fees paid by all trade aggressors, and which makers the fees were paid to.
-  repeated TradingFeesGenerated trading_fees_generated = 10;
+  // Total maker fees received by the maker side.
+  repeated PartyAmount total_fees_paid_and_received = 11;
```

Comment:
The name implies fees paid, but the docs only refer to fee revenue received, which one is correct?